seed/java-sdk/bytes/src/main/java/com/seed/bytes/core/FileStream.java

```diff
@@ -0,0 +1,57 @@
+package com.seed.bytes.core;
+
+import java.io.InputStream;
+import java.util.Objects;
+import okhttp3.MediaType;
+import okhttp3.RequestBody;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Represents a file stream with associated metadata for file uploads.
+ */
+public class FileStream {
```

Comment:
we don't actually need this for bytes requests, going to delete unless anyone votes I keep it around

Reply:
awesome, may be useful to keep around to make it better for multipart (just as a uncopied utility)